pkg/client/tables.go

```diff
@@ -343,3 +351,104 @@ func (c *Client) DeleteTableEntry(ctx context.Context, entry *p4_v1.TableEntry)
 
 	return c.WriteUpdate(ctx, update)
 }
+
+func (c *Client) NewActionProfileMemberEntry(
+	table string,
+	memberId uint32,
```

Comment:
nit: please use `memberID` and `groupID` instead of `memberId` and `groupId` (https://github.com/golang/go/wiki/CodeReviewComments#initialisms)